.github/actions/argus-builder/build-prep/action.yml

```diff
@@ -1,18 +1,17 @@
-name: argus-docker-build-prep
+name: argus-builder/build-prep
 description: Prepare for building a Docker Image for Argus
 
 inputs:
+  github_token:
+    description: The GitHub token for authenticating with the GitHub API
+    required: true
+  images:
+    description: 'JSON object specifying the images to build'
+    required: true
   path_filters:
     description: 'Glob patterns to match against changed files in the repository, comma delimited'
     required: false
     default: '**/*'
-  path_filters_base:
```

Comment:
I think we'll need to do a major version bump to remove this

Reply:
New plan, doing a major bump so I can clean this up since a major bump is going to be needed for the next changes to the `manifest-update` step